nuget/targets/netstandard/Microsoft.ML.OnnxRuntime.Extensions.props

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <AdditionalIncludeDirectories>$(MSBuildThisFileDirectory)../../build/native/include/;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
for desktop, we don't have any header files needs to be published.

Reply:
This nuget package provides a native binary so if someone wants to use that directly they need the headers.

This is consistent with what the ORT native nuget package includes.

Reply:
removed